src/InputContainer.jsx

```diff
@@ -18,16 +18,16 @@ export default function InputContainer() {
     listAddress: state.listAddress,
   }));
 
-  function handleChangeName(event) {
-    dispatch(updateName(event.target.value));
-  }
-
-  function handleChangeCategory(event) {
-    dispatch(updateCategory(event.target.value));
-  }
-
-  function handleChangeAddress(event) {
-    dispatch(updateAddress(event.target.value));
+  function handleChangeInput({ target: { name, value } }) {
```

Comment:
```suggestion
  function handleChangeInput({ name, value }) {
    ....
    })
```

container 에서는 event 객체 대신, 필요한 정보만 받아서 의존성을 끊으면 좋을거같아요 :) 그러려면 Input component 에서  아래와같이 작업해주면 되겠죠? (네이밍은 급하게 지은거라, 수정이 필요하면 수정해주세요!! )

```
function Input({ ... onChange }) {
  function handleChange(e) {
    const { name, value} = e.target;

    onChange({name, value});
  }

  return (
    ....
  )
}
```